### ISSUE TITLE
consider using block level coverage

### ISSUE THREAD

**Issue by connorjclark** [GoogleChrome/lighthouse#11407]
ref https://github.com/GoogleChrome/lighthouse/blob/8dd649aa30553b6862229fcee57d8c2039500b70/lighthouse-core/gather/gatherers/js-usage.js#L35

https://bugs.chromium.org/p/chromium/issues/detail?id=1120879

Might be (much?) slower. Waiting for yang to confirm.

**Comment by patrickhulce** [comment]
FWIW I'm not sure we need to do this given how frequently we end up flagging users for unused JavaScript already. If anything I think that audit could be a little *more* lenient to give more room for code used on interaction. With that lens, features that (might) slow down the run and make us *less* lenient at the same time don't seem super appealing

**Comment by connorjclark** [comment]
Good point, but I'd rather be more accurate and vary our threshold to modulate our leniency.

**Comment by connorjclark** [comment]
Speed is better than accuracy for Lighthouse. Users debugging unused javascript should use the DevTools coverage tool.
